src/components/PaymentRequestActionMenu/PaymentRequestActionMenu.tsx

```diff
@@ -0,0 +1,82 @@
+import { cva } from 'class-variance-authority';
+import * as DropdownMenu from '@radix-ui/react-dropdown-menu';
+import menuIcon from '../../assets/images/nf_menu.svg';
+import copyIcon from '../../assets/images/nf_copy.svg';
+import linkIcon from '../../assets/images/nf_link.svg';
+import trashIcon from '../../assets/images/nf_trash.svg';
+
+const actionItem = cva(
+  'group text-xs leading-none rounded-1 flex items-center h-6 py-4 pr-4 relative select-none outline-none cursor-pointer data-[highlighted]:text-greyText',
+  {
+    variants: {
+      intent: {
+        neutral: [],
+        negative: ['text-red-700'],
+      },
+    },
+    defaultVariants: {
+      intent: 'neutral',
+    },
+  },
+);
+
+interface PaymentRequestActionMenuProps {
+  onDuplicate?: () => void;
+  onCopyLink?: () => void;
+  onDelete?: () => void;
+}
+
+interface PaymentRequestActionMenuItemContentProps {
+  label: string;
+  iconSource: string;
+}
+
+const PaymentRequestActionMenuItemContent = ({ label, iconSource }: PaymentRequestActionMenuItemContentProps) => {
+  return (
+    <>
+      <div className="pl-4 pr-2">
+        <img src={iconSource} alt={label} />
+      </div>
+      {label}
+    </>
```

Comment:
Maybe Pablo I. would leave this message, but I feel like the items needs a kind of hover to make them feel better.

Do you know if there's any design about this?

Reply:
It does have a hover effect. The class is assigned to the parent element.